operator/config/default/manager_webhook_patch.yaml

```diff
@@ -1,3 +1,5 @@
+# The patch is only required for non-OLM-based installations because OLM creates and mounts these volumes itself.
```

Comment:
```suggestion
# The patch is only required for non-OLM-based installations because OLM creates volumes and volumeMounts itself dynamically at runtime.
```